crates/telio-model/src/features.rs

```diff
@@ -369,10 +369,13 @@ impl FeatureEndpointProvidersOptimization {
     }
 }
 
-/// Turns on connection resets upon VPN server change
+/// Feature config for firewall
 #[derive(Default, Clone, Copy, Debug, PartialEq, Eq, Deserialize)]
-#[serde(transparent)]
-pub struct FeatureBoringtunResetConns(pub bool);
+pub struct FeatureFirewall {
```

Comment:
I started adding tests in a different way in here for multicast. What do you think of creating similar tests for this new firewall feature config block?

Reply:
Can you elaborate more on different way ?

Reply:
https://github.com/NordSecurity/libtelio/blob/main/crates/telio-model/src/features.rs#L1255 basically a module for `FeatureFirewall` that checks that it is parsed correctly